### DISCUSSION TITLE
Skeletons into Wither Skeletons

### DISCUSSION THREAD

**Discussion by MrRaposinha** [PurpurMC/Purpur#970] [__Completed__] [answered]
Add a config option to enable a new mechanic that is basically on right clicking a skeleton with a wither rose will turn it into a wither skeleton.

**Comment by YouHaveTrouble** [comment]
Sounds like a job for a plugin tbh

**Reply by SageSphinx63920** [reply]
Yeah

**Reply by katekerllenevich** [reply]
I would argue this is [just the job for purpur](https://github.com/PurpurMC/Purpur/blob/ver/1.18.2/patches/server/0038-Cows-eat-mushrooms.patch)

**Reply by SageSphinx63920** [reply]
Thx

**Comment by katekerllenevich** [comment, marked as answer]
Implemented in 6ce9e6e4
